crates/ruff_python_formatter/resources/test/fixtures/ruff/newlines.py

```diff
@@ -249,6 +249,60 @@ def y():
     print()
 
 
+if True:
+    def a():
+        return 1
+else:
```

Comment:
Can we add a few tests related to suppression comments? 

I'm worried that we now add empty lines even if the entire suite is suppressed.

Reply:
I added one, is that what you had in mind?

Reply:
Looks good, thanks